demisto_sdk/commands/create_id_set/tests/layout-to-test.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
Put this file inside test_data folder. Such that: `demisto_sdk/commands/create_id_set/tests/test_data/layout-to-test.json`